packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -11,13 +11,57 @@ export default class QuanticSort extends LightningElement {
   sort;
   /** @type {import("coveo").Unsubscribe} */
   unsubscribe;
+  /** @type {HTMLElement} */
+  combobox;
+  /** @type {HTMLElement} */
+  relevancyOption;
+  /** @type {HTMLElement} */
+  newestOption;
+  /** @type {HTMLElement} */
+  oldestOption;
+  /** @type {HTMLElement} */
+  checkIcon;
+  /** @type {HTMLElement} */
+  sortIcon;
+  /** @type {HTMLElement} */
+  selectedOption;
+  /** @type {string} */
+  sortMethod;
 
   connectedCallback() {
     registerComponentForInit(this, this.engineId);
   }
 
   renderedCallback() {
     initializeWithHeadless(this, this.engineId, this.initialize.bind(this));
+    if(!this.combobox){
+      this.combobox = this.template.querySelector('.slds-dropdown-trigger_click');
+    }
+    if(!this.relevancyOption){
+      this.relevancyOption = this.template.querySelectorAll('.slds-listbox__option')[0];
```

Comment:
Instead of relying on the option index, you should define CSS classes to put on the well-known elements.

For instance, we could have `sort__option_relevancy`

Reply:
Sounds good. I will do the same for the icon elements since I am using the index there as well